include/RE/RTTI.h

```diff
@@ -111,30 +130,8 @@ namespace RE
 		};
 		static_assert(sizeof(CompleteObjectLocator) == 0x14);
 
-		struct BaseClassDescriptor
-		{
-		public:
-			enum class Attribute
-			{
-				kNone = 0,
-				kNotVisible = 1 << 0,
-				kAmbiguous = 1 << 1,
-				kPrivate = 1 << 2,
-				kPrivateOrProtectedBase = 1 << 3,
-				kVirtual = 1 << 4,
-				kNonPolymorphic = 1 << 5,
-				kHasHierarchyDescriptor = 1 << 6
-			};
-
-			// members
-			RVA<TypeDescriptor>						   typeDescriptor;	   // 00
-			std::uint32_t							   numContainedBases;  // 04
-			PMD										   pmd;				   // 08
-			stl::enumeration<Attribute, std::uint32_t> attributes;		   // 14
-		};
-		static_assert(sizeof(BaseClassDescriptor) == 0x18);
-
 		void DumpTypeName(void* a_obj);
+		void DumpClassHier(void* a_obj);
```

Comment:
You can inspect a class's hierarchy in ida. I don't see the need for this at runtime.

Reply:
I don't have IDA, and no other open source reverse engineering utilities actually allow for the importing of large number of type definitions making this my only option.

Reply:
I'd suggest you hand roll a utility for this then.